tests/test_attendeeprofiles.py

```diff
@@ -0,0 +1,53 @@
+
+"""
+TODO(artcz)(2019-01-09) - Filename WIP
```

Comment:
Are you leaving this as a todo? :)

Reply:
I'd rather have test files named after use cases, but this one doesn't belong to any other existing use case (say 'invocing' or 'cfp') and I didn't want to create something for a single test that wouldn't be obvious why this test is there.

I guess this will finally belong to something like `test_attendees.py` (or maybe something related to the speakers, since the main reason for fixing this was fixing old information from previous conference)

Consider it a placeholder for now. ;)